### PR TITLE
fix deadlinks

### PR DESCRIPTION
**Pull request by willhoy**
links need to include /docs

*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,19 +67,19 @@ Comprehensive documentation about BlazingMQ can be found [here](https://bloomber
...
-[companion](https://bloomberg.github.io/blazingmq/getting_started/more_fun_with_blazingmq)
+[companion](https://bloomberg.github.io/blazingmq/docs/getting_started/more_fun_with_blazingmq)
 article, readers can learn about some intermediate and advanced features of BlazingMQ and see them in action.
 
 ---
 
 ## Installation
 
-[This](https://bloomberg.github.io/blazingmq/configuration-and-deployment/deployment)
+[This](https://bloomberg.github.io/blazingmq/docs/configuration-and-deployment/deployment)
```
Comment (hallfox) [inline README.md:82]:
```suggestion
[This](https://bloomberg.github.io/blazingmq/docs/installation/deployment/)
```

Comment (678098) [inline README.md:82]: 404 on this link

**Review by 678098 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (willhoy) [inline README.md:82]:
```suggestion
[This](https://bloomberg.github.io/blazingmq/docs/installation/configuration/)
```

**Review by hallfox — APPROVED** [review]

**Review by mlongob — APPROVED** [review]
LGTM

**Pull request merged**